style.css

```diff
@@ -2,19 +2,29 @@ html,
 body {
   font-family: "Poppins", sans-serif;
   font-style: normal;
-  background: #e5e5e5;
+  background: #fff;
 }
 
 header {
   position: absolute;
   height: 718px;
+  width: 375px;
   left: 0;
   top: 0;
   right: 0;
   padding-left: 24px;
   padding-right: 24px;
-  padding-top: 8px;
-  background: #fff;
+  padding-top: 20px;
+}
+
+section {
+  padding-left: 24px;
+  padding-right: 24px;
+  padding-top: 120px;
+  background-image: url("Images/headershapes-mobile.svg");
```

Comment:
- [ ] Great job adding the right background image. But your URL is incorrect. You are using a capital `I` instead of a small letter `i`. Kindly fix this. RIght URL `images/headershapes-mobile.svg`.